pkgs/tools/misc/gtklp/default.nix

```
@@ -0,0 +1,55 @@
+{ stdenv, fetchurl
+, autoconf, automake, libtool, pkgconfig, fetchpatch
+, gtk2, glib, cups
+, gettext, openssl
+}:
+
+stdenv.mkDerivation rec {
+  pname = "gtklp";
+  version = "1.3.4";
+
+  src = fetchurl {
+    url = "mirror://sourceforge/${pname}/${pname}-${version}.src.tar.gz";
+    sha256 = "1arvnnvar22ipgnzqqq8xh0kkwyf71q2sfsf0crajpsr8a8601xy";
+  };
+
+  nativeBuildInputs = [
+    pkgconfig
+  ];
+
+  buildInputs = [
+    autoconf
+    automake
+    cups
+    gettext
+    glib
+    gtk2
+    libtool
+    openssl
+  ];
+
+  patches = [ ../gtklp/formatsec.patch ];
+
+  preConfigure = ''
+    substituteInPlace include/defaults.h --replace "netscape" "firefox"
+    substituteInPlace include/defaults.h --replace "http://localhost:631/sum.html#STANDARD_OPTIONS" \
+                                                   "http://localhost:631/help/"
+
+    # bypass stale configure script
+    aclocal -I m4
+    autoheader --force
+    autoconf --force
+    automake --add-missing --copy --include-deps --foreign --force-missing
+  '';
+
+  preInstall = ''
+    install -D -m0644 -t $out/share/doc AUTHORS BUGS ChangeLog README TODO USAGE
```

I think TODO is not relevant.

```suggestion
    install -D -m0644 -t $out/share/doc AUTHORS BUGS ChangeLog README USAGE
```